model/src/pyrenew/model/hospitalizations.py

```diff
@@ -87,27 +87,27 @@ def __init__(
             gen_int=gen_int,
             I0=I0,
             latent_infections=latent_infections,
-            observed_infections=DeterministicVariable((0,)),
+            observation_process=None,
             Rt_process=Rt_process,
         )
 
         HospitalizationsModel.validate(
-            latent_hospitalizations, observed_hospitalizations
+            latent_hospitalizations, observation_process
         )
 
         self.latent_hospitalizations = latent_hospitalizations
-        self.observed_hospitalizations = observed_hospitalizations
+        self.observation_process = observation_process
```

Comment:
Don't we need to overwrite with a `NullObservationProcess` or similar if set to `None`?